#
# patch created with `svn diff -r53 . > patches/3com-oc-remote812.patch`
# on Wed Jun  6 15:07:45 CEST 2007
# applied on r53
# 
# This patch was send to me by The_Beast on IRC.
#
Index: ddclient
===================================================================
--- ddclient	(revision 53)
+++ ddclient	(working copy)
@@ -162,7 +162,7 @@
     '3com-oc-remote812'      => {
     				  'name' => '3com OfficeConnect Remote 812',
                                   'url'  => '/callEvent',
-                                  'skip' => '.*IP address in use',     
+                                  'skip' => '.*LOCAL',
                                 },
     'e-tech'		     => {
 				  'name' => 'E-tech Router',
